mmdet3d/datasets/s3dis_dataset.py

```diff
@@ -51,8 +51,8 @@ class S3DISDataset(Det3DDataset):
     METAINFO = {
         'classes': ('table', 'chair', 'sofa', 'bookcase', 'board'),
         # the valid ids of segmentation annotations
-        'seg_valid_class_ids': tuple(range(13)),
-        'seg_all_class_ids': tuple(range(14))  # possibly with 'stair' class
+        'seg_valid_class_ids': (7, 8, 9, 10, 11),
+        'seg_all_class_ids': tuple(range(1, 14))  # possibly with 'stair' class
```

Comment:
Did you check this modification? If the `seg_all_class_ids` starts with 0, then the `seg_valid_class_ids` maybe start with 8?  To keep consistency with , maybe  `'seg_all_class_ids' : tuple(range(14))`. Please check it again.

Reply:
I have checked the modification. Both of `tuple(range(1, 14))` and `tuple(range(14))` seem no effect on the result. 
For `'seg_all_class_ids': tuple(range(1, 14))`, the `seg_label_mapping` is `array([5, 5, 5, 5, 5, 5, 5, 0, 1, 2, 3, 4, 5, 5])`.
For `'seg_all_class_ids': tuple(range(14))`, the `seg_label_mapping` is `array([5, 5, 5, 5, 5, 5, 5, 0, 1, 2, 3, 4, 5, 5, 5])`.
Both will map the seg class_id start with 0.

Reply:
Thanks for your check. Are there any influences between `'seg_valid_class_ids': (7, 8, 9, 10, 11)` and `'seg_valid_class_ids': ( 8, 9, 10, 11, 12)`

Reply:
We should only make sure `len(seg_all_class_ids)>=13` and that will not influence other `seg_valid_class_ids` settings.